plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveHadoop.java

```diff
@@ -58,7 +69,14 @@ private HiveHadoop(
                 filesToMount,
                 envVars,
                 network,
+                waitStrategy,
                 startupRetryLimit);
+        try {
+            hostAddress = InetAddress.getLocalHost().getHostAddress();
```

Comment:
That's not an attribute of the container, so shouldn't be a field in `HiveHadoop` class